libs/partners/cohere/langchain_cohere/chat_models.py

```diff
@@ -85,10 +86,10 @@ def get_cohere_chat_request(
         "chat_history": [
             {"role": get_role(x), "message": x.content} for x in messages[:-1]
         ],
+        **additional_kwargs, # params below will overwrite these values
         "documents": documents,
-        "connectors": maybe_connectors,
+        "connectors": connectors,
         "prompt_truncation": prompt_truncation,
```

Comment:
we still need **kwargs, I think! Otherwise a lot of the parameters are dropped.

Reply:
The reason for passing in `**kwargs` in the return is exactly to forward any Cohere SDK params to the API. However those params are coming from `**additional_kwargs` (aka `documents`, `connectors`, `prompt_truncation`) that are pre-processed in this occasion), I believe `**kwargs` never contained those parameters, and hence removed.

Will defer to LC team's take on this as well however 🙏.

Reply:
Comments are deprecated. tldr: move `kwargs` back in.